roles/letsencrypt/tasks/update.yml

```diff
@@ -0,0 +1,23 @@
+- name: Install letsencrypt challenges
+  copy:
+    dest: "{{ letsencrypt_webroot }}/{{ item.value['http-01']['resource'] }}"
+    content: "{{ item.value['http-01']['resource_value'] }}"
+  with_dict: "{{ challenge_data }}"
+
+- name: Update letsencrypt certificates
+  letsencrypt:
```

Comment:
I was using this as a test, letsencrypt provides a staging server that you can fetch certs from that doesn't annoy everyone when you request them a lot or delete things. So production was whether to use the real server or the staging server. 

I had keyed the role load off whether cn was specified, but we could do something more obvious if you like?